chef_master/source/server_overview.rst

```diff
@@ -301,7 +301,42 @@ Use the following configuration settings in the chef-server.rb file to configure
    * - ``postgresql['vip']``
      - Required when ``postgresql['external']`` is set to ``true``. The virtual IP address. The host for this IP address must be online and reachable from the Chef Infra Server via the port specified by ``postgresql['port']``. Set this value to the IP address or hostname for the machine on which external PostgreSQL is located when ``postgresql['external']`` is set to ``true``.
 
+Optional Settings
++++++++++++++++++++++++++++++++++++++++++++++++++++++
+The following optional settings are required when configuring External PostgreSQL on Microsoft Azure:
+
+.. list-table::
+   :widths: 200 300
+   :header-rows: 1
+
+   * - Setting
+     - Description
+   * - ``bookshelf['sql_connection_user']``
+     - The PostgreSQL user name in ``'<username@hostname>'`` format (e.g. ``'bookshelf@my_postgresql.postgres.database.azure.com'``) used for making PostgreSQL queries, where ``username`` would normally equal the value of ``bookshelf['sql_user']`` (default ``'bookshelf'``). Default value: ``nil``.
```

Comment:
A nil value means the system uses the string 'bookshelf', right? So if they were using azure they'd need to fully define this.

Reply:
A nil value for this setting (bookshelf['sql_connection_user']) means the connection will be made using bookshelf['sql_user'], but the default value for **_that_** is the string 'bookshelf'.  Let me know if I communicated that well enough or if you would reword it.